dev_guide/pod_disruption_budget.adoc

```diff
@@ -0,0 +1,65 @@
+[[dev-guide-pod-disruption-budget]]
+= Pod Distribution Budget
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+A _Pod Disruption Budget_ is part of the
+link:http://kubernetes.io/docs/admin/disruptions/[Kubernetes] API, which can be
+managed with `oc` commands like other
+xref:../cli_reference/basic_cli_operations.adoc#object-types[object types]. They
+allow specifying safety constraints on pods during operations, such as draining a
+node for maintenance.
+
+ifdef::openshift-enterprise[]
+[NOTE]
+====
+As of {product-title} 3.4, Pod Disruption Budget is a feature in Technology
+Preview, available only for users with *cluster-admin* privileges.
+====
+endif::[]
+
+[[creating-a-poddisruptionbudget]]
+== Creating a Pod Disruption Budget
+
+A `PodDisruptionBudget` configuration consists of the following key parts:
+
+* A label selector, which is a label query over a set of pods.
+* An availability level, which specifies the minimum number of pods that must be
+ available simultaneously.
+
+The following is an example of a `PodDisruptionBudget` resource:
+
+====
+[source,yaml]
+----
+apiVersion: policy/v1alpha1 <1>
+kind: PodDisruptionBudget
+metadata:
+  name: foo
+spec:
+  selector:  <2>
+    matchLabels:
+      foo: bar
+  minAvailable: 2  <3>
+----
+
+<1> `PodDisruptionBudget` is part of the `policy/v1alpha` API group.
+<2> A label query over a set of resources. The result of `matchLabels` and
+ `matchExpressions` are ANDed.
```

Comment:
"conjoined bitwise" or "conjoined with a bitwise AND operator" seem to be correct phrasing based on some resources @adellape pointed me to.